src/components/wallet/earn/CreateOfferForm.vue

```diff
@@ -818,32 +816,43 @@ export default class CreateOfferForm extends Vue {
         await this.$store.dispatch('Assets/updateUTXOs')
         await this.$store.dispatch('Signavault/updateTransaction')
     }
+    get isLoading() {
+        return this.txState === TxState.waiting
+    }
     async waitTxConfirm(
         txId: string,
         addresses: {
             address: string
         }[],
-        timestamp: number
+        timestamp: number,
+        restricted: boolean
     ) {
-        let status = await ava.PChain().getTxStatus(txId)
+        let response: string | GetTxStatusResponse = await ava.PChain().getTxStatus(txId)
+        let status = (response as GetTxStatusResponse).status
         if (status === 'Unknown' || status === 'Processing') {
             // if not confirmed ask again
             setTimeout(() => {
-                this.waitTxConfirm(txId, addresses, timestamp)
+                this.waitTxConfirm(txId, addresses, timestamp, restricted)
             }, 500)
             return false
         } else if (status === 'Dropped') {
             // If dropped stop the process
             this.txState = TxState.failed
             return false
         } else {
-            this.$store.dispatch('Platform/addAllowedAddresses', {
-                depositOfferID: txId,
-                allowedAddresses: addresses,
-                timestamp,
+            if (restricted)
```

Comment:
why do we show it only when it's restricted?

Reply:
the notification always displayed the if condition applied only for sending a request to signavault